$NetBSD: patch-ah,v 1.1.1.1 2007/02/09 19:57:48 drochner Exp $

The MAX macro is not defined on all systems (e.g. solaris)
 
--- lib/rtp/rtp.c.orig	2006-03-20 18:08:24.000000000 -0500
+++ lib/rtp/rtp.c
@@ -61,6 +61,10 @@
 
 #include "rtp.h"
 
+#ifndef MAX
+#define MAX(a,b) ((a) > (b) ? (a) : (b))
+#endif
+
 typedef struct {
   uint32_t key;   /* Original allocation number   */
   uint32_t size;  /* Size of allocation requested */
